src/sysext/sysext.c

```diff
@@ -2219,7 +2219,6 @@ static int parse_merge_parameters(sd_varlink *link, sd_json_variant *parameters,
         };
 
         assert(link);
-        assert(parameters);
```

Comment:
removing this is unnecessary, we always pass a non-NULL parameter here, so the assert() should never trigger?